src/_includes/docs/swift-package-manager/how-to-enable-disable.md

```diff
@@ -0,0 +1,62 @@
+## How to enable Swift Package Manager
+
+Flutter's Swift Package Manager feature is disabled by default.
+To enable it:
+
+1. Switch to Flutter's `master` channel:
+
+   ```sh
+   flutter channel master
+   flutter upgrade
+   ```
+
+2. Enable the Swift Package Manager feature:
+
+   ```sh
+   flutter config --enable-swift-package-manager
+   ```
+
+Running an app using the Flutter CLI will migrate it to add Swift Package
+Manager support.
+This makes your project download the Swift packages that
+your Flutter plugins depend on.
+
+:::note
+Flutter will fallback to CocoaPods for dependencies that do not support Swift
+Package Manager yet.
+:::
+
+## How to disable Swift Package Manager
```

Comment:
Ideally app developers enable SwiftPM and never disable it.

However, plugin developers need to enable/disable SwiftPM to test their plugin works with both scenarios.